blog/_posts/2021-10-08-old-school-remote-dev-clusters.md

```diff
@@ -0,0 +1,163 @@
+---
+slug: "old-school-remote-dev-clusters"
+date: 2021-10-07
+author: nick
+layout: blog
+title: "Setting up a Remote Dev Environment When You Hate the Cloud"
```

Comment:
Not sure we want our material associated with hateful feelings. Lian suggested 'struggle' or something similar.

Reply:
alternatively: "When you'd rather run things under your desk" 😅